test/stubs/dss_stub.c

```diff
@@ -0,0 +1,10 @@
+/*
+ * dss_stub.c
+ *
+ * Stub for the dss
+ */
+
+
+void dss() {
+  return;

```

Comment:
If you are going to leave this empty, please add a comment like: `intentionally empty` so that it doesn't look like an oversight or a mistake.